kustomize/network/calico-etcd/kustomization.yaml

```diff
@@ -19,4 +20,4 @@ secretGenerator:
 patchesStrategicMerge:
   - patch_image_name.yaml
 
-namespace: network-test-90fa3885
+namespace: network-test-f22e8ea9
```

Comment:
if it's a temporary name, shouldn't we generate it during testing? hardcode the value means you will change it every time when creating a pr .

Reply:
it is a patch and will change each time. it is not hardcoded and it will be used by test or publish